hasher-matcher-actioner/webapp/src/components/ContentMatchPane.jsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+import React, {useEffect, useState} from 'react';
+import {Link} from 'react-router-dom';
+import PropTypes from 'prop-types';
+import {Container, Row, Col} from 'react-bootstrap';
+import {fetchHash, fetchImage} from '../Api';
+import {BlurUntilHoverImage} from '../utils/ImageUtils';
+
+/**
+ * For a content key that matched a signal, show a summary and link to other
+ * routes that show more information.
+ */
+export default function ContentMatchPane({contentId, signalId, signalSource}) {
+  const [hashDetails, setHashDetails] = useState(null);
+  const [img, setImage] = useState(null);
+
+  useEffect(() => {
+    fetchHash(contentId).then(hash => {
+      setHashDetails(hash);
+    });
+  }, [contentId]);
+
+  useEffect(() => {
+    fetchImage(contentId)
+      .then(result => {
+        setImage(URL.createObjectURL(result));
+      })
+      .catch(() => {
+        setImage(
+          'https://image.shutterstock.com/image-vector/mission-failed-text-on-red-260nw-1633188751.jpg',
+        );
+      });
+  }, [contentId]);
+
+  return (
+    (hashDetails === null && <p>Loading...</p>) || (
+      <Container>
+        <Row>
+          <Col className="p-4">
+            <BlurUntilHoverImage src={img} />
+          </Col>
+        </Row>
+        <Row>
+          <Col md={3} className="p-4">
+            Signal
+          </Col>
+          <Col className="p-4">
+            <Link to={`?signalId=${signalSource}|${signalId}`}>
+              View Matches for Signal
+            </Link>
+          </Col>
+        </Row>
+        <Row>
+          <Col md={3} className="p-4">
+            Content
+          </Col>
+          <Col className="p-4">
+            <Link to={`?contentId=images/${contentId}`}>
```

Comment:
As it is currently structured there is no way to get to the "MatchDetails" (ContentDetails) subpage, can you add a 2nd link/button here for this? 
Otherwise the only way to get there is manually setting the URL or right after an upload (which is worse than not at all).